packages/cli/src/lib/cmds/init.ts

```diff
@@ -91,19 +91,26 @@ async function confirmTwaConfig(twaManifest: TwaManifest): Promise<TwaManifest>
       type: 'input',
       message: 'Android Package Name (or Application ID):',
       default: twaManifest.packageId,
-      validate: notEmpty,
+      validate: async (input): Promise<boolean> => {
+        if (!util.validatePackageId(input)) {
+          throw new Error('Invalid Application Id. Check requiements at ' +
+              'https://developer.android.com/studio/build/application-id');
+        }
+        return true;
+      },
     }, {
       name: 'keyPath',
       type: 'input',
       message: 'Location of the Signing Key:',
       default: twaManifest.signingKey.path,
-      validate: notEmpty,
+      validate: async (input): Promise<boolean> =>
+        notEmpty(input, 'KeyStore location cannot be empty'),
     }, {
       name: 'keyAlias',
       type: 'input',
       message: 'Key name:',
       default: twaManifest.signingKey.alias,
-      validate: notEmpty,
+      validate: async (input): Promise<boolean> => notEmpty(input, 'Key alias cannot be empty'),
```

Comment:
Could you just pass the name to `notEmpty` and have `notEmpty` take care of the "cannot be empty" part? (As it is, for half of these you say "cannot be empty" and the other half you have "can't be empty".

Reply:
done